src/xrCore/xr_ini.cpp

```diff
@@ -451,13 +451,20 @@ void CInifile::Load(IReader* F, pcstr path, allow_include_func_t allow_include_f
             R_ASSERT(path && path[0]);
             if (_GetItem(str, 1, inc_name, '"'))
             {
-                xr_fs_nostrlwr(inc_name); // compensate removed xr_strlwr on path on Linux, etc
-
+                xr_fs_strlwr(inc_name); // compensate removed xr_strlwr on path on Linux, etc
```

Comment:
That changes the behaviour on Windows!

Reply:
Why? Windows is not case-sensitive in paths. If you do not change the case of characters in this place, will nothing change for windows? So?

Reply:
Because strings comparison inside the engine is case-sensitive.

Reply:
The `xr_fs_nostrlwr` function does not change case on Windows, and `xr_fs_strlwr` does not change case on Linux. So maybe remove the `xr_fs_strlwr` function here?

Reply:
I think call of `xr_fs_strlwr` unnecessary

Reply:
Removed